src/client.ts

```diff
@@ -302,22 +307,26 @@ export class APIClient {
   }
 }
 
-export function parseNumRequests(str) {
-  let retValue;
-  try {
-    retValue = parseInt(str);
-  } catch (err) {
-    throw new IntegrationProviderAuthenticationError({
-      cause: err,
-      endpoint: "client.parseNumRequests",
-      status: err.status,
-      statusText: "There was a problem parsing the NUM_REQUESTS config field.",
-    });
+export function stopFetchingRequests(requests, lastExecutionTime) {
```

Comment:
I think the linter should be requiring argument and return typing. Does your environment have the Husky hooks installed in `.git/hooks/pre-commit`?

Reply:
Yes, they are there. Don't know why it didn't complain on these, but I updated the code with types.